packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -96,7 +96,7 @@ export default function NavigationInnerBlocks( {
 		isSelected ||
 		( isImmediateParentOfSelectedBlock && ! selectedBlockHasDescendants );
 
-	const placeholder = useMemo( () => <PlaceholderPreview />, [] );
+	const placeholder = PlaceholderPreview;
```

Comment:
This appeared to be causing the component to be rendered when it didn't need to be.

I'm not sure this component is rendered often enough to warrant a memo here but I could be wrong.